src/main/java/com/linkurlshorter/urlshortener/link/LinkController.java

```diff
@@ -241,6 +244,16 @@ public ResponseEntity<LinkStatisticsResponse> getLinksStatsForUser() {
         return ResponseEntity.ok(new LinkStatisticsResponse(stats, "ok"));
     }
 
+    @GetMapping("/all-links")
+    @Operation(summary = "Get all links")
+    public ResponseEntity<LinkInfoResponse> getAllLinks() {
+        List<LinkInfoDto> links = linkService.findAllActive()
+                .stream()
+                .map(linkDtoMapper::mapLinkToDto)
+                .collect(Collectors.toList());
```

Comment:
Better to replace this usage of 'Stream.collect(Collectors.toList())' with 'Stream.toList()'